test_suite/lris_blue_long_400_3400_d560.pypit

```diff
@@ -3,19 +3,19 @@
 # Change the default settings
```

Comment:
I've deleted the .pypit files, since they need not be here anymore, but I've kept the rest of these files, as they do not appear to be in the development suite, and might be useful in the future. I'll raise the issue and leave it up to you to decide their fate  ;-)